src/components/Header/CreateNewChatMobile.tsx

```diff
@@ -17,7 +17,7 @@ export const CreateNewChatMobile = () => {
   return (
     <Tooltip isTriggerClickable tooltip={t('New conversation')}>
       <div
-        className="flex border-r border-gray-300 p-3 dark:border-gray-900 md:hidden"
+        className="flex border-r border-primary p-3 md:hidden"
```

Comment:
```suggestion
        className="flex border-r border-tertiary p-3 md:hidden"
```
![image](https://github.com/epam/ai-dial-chat/assets/20163971/7db1ef82-b5eb-41e8-b055-ee5c4004744d)